Package@swift-5.6.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.6
```

Comment:
until we hear more back from @fabianfett, let go with the "old files style", i.e make Package.swift required 5.6 and add files for each one of the old version we support.